src/case-worker-ref-data/components/case-worker-details/case-worker-details.component.html

```diff
@@ -0,0 +1,49 @@
+<form #excelForm=ngForm (ngSubmit)="onSubmit(Excel)">
+    <div class="hmcts-width-container">
+        <main class="govuk-main-wrapper " id="main-content" role="main">
+            <div class="govuk-error-summary" *ngIf="errorDesc" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+                <h2 class="govuk-error-summary__title" id="error-summary-title">
+                    There is a problem
+                </h2>
+                <div class="govuk-error-summary__body">
+                    <ul class="govuk-list govuk-error-summary__list">
+                        <li>
+                            
+                                <a href="#uploadarea">{{errorDesc}}</a>
+                            
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <h1 class="govuk-heading-l">Caseworker details</h1>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-three-quarters">
+                    <p class="govuk-body">You can add, update or delete existing caseworker details. For example, their name, role and location.</p>
+                    <p class="govuk-body">You can do this by:</p>
+                    <ol class="govuk-body">
+                        <li>Opening your existing caseworker details spreadsheet. You can <a href="#"> find out how to create a template</a>.</li>
+                        <li>Updating the caseworker details.</li>
+                        <li>Selecting and uploading the updated template.</li>
+                    </ol>
+                    <p class="govuk-body">You’ll then be able to assign tasks to your colleagues using the Manage cases application.</p>        
+                    <div class="govuk-form-group" [ngClass]="{ 'govuk-form-group--error': errorDesc }" id="uploadarea">
+                        <label class="govuk-label" for="file-upload-1"><strong>
+                            Upload file</strong>
+                        </label>
+
+                        
+                            <span id="" *ngIf="errorDesc" class="govuk-error-message">
+                              <span class="govuk-visually-hidden">Error:</span> {{errorDesc}}
+                            </span>
+                    <input class="govuk-file-upload" #Excel id="file-upload-1" name="file-upload-1" type="file" accept=" .xls, .xlsx">
```

Comment:
Indenting is off on this line. The whole file should be 2 space indenting, but this one is further back than it should be.